modules/apps/item-selector/item-selector-taglib/src/main/java/com/liferay/item/selector/taglib/internal/util/GroupItemSelectorProviderRegistryUtil.java

```diff
@@ -70,8 +68,15 @@ public static Set<String> getGroupItemSelectorProviderTypes() {
 		return types;
 	}
 
-	@Activate
-	protected void activate(BundleContext bundleContext) {
+	private static final ServiceTrackerMap<String, GroupItemSelectorProvider>
```

Comment:
@MrJohn1911 The service tracker map is now static final, so the null check in the methods above are no longer needed.

```
		if (_serviceTrackerMap == null) {
			return null;
		}
```